src/components/pages/rules/flyouts/ruleNew/ruleNew.js

```diff
@@ -131,48 +211,58 @@ export class RuleNew extends LinkedComponent {
-                      <Btn svg={svgs.trash} onClick={this.deleteCondition(idx)}>Delete</Btn>
-                    }
                     <FormGroup>
                       <FormLabel isRequired="true">{t(`rulesFlyout.condition.field`)}</FormLabel>
                       <FormControl
                         type="select"
                         className="long"
                         placeholder={t(`rulesFlyout.condition.fieldPlaceholder`)}
-                        link={condition.field} />
+                        link={condition.fieldLink}
+                        options={this.state.fieldOptions}
+                        clearable={false}
+                        searchable={true} />
                     </FormGroup>
                     <FormGroup>
                       <FormLabel isRequired="true">{t(`rulesFlyout.condition.calculation`)}</FormLabel>
                       <FormControl
                         type="select"
                         className="long"
                         placeholder={t(`rulesFlyout.condition.calculationPlaceholder`)}
-                        link={condition.calculation} />
+                        link={condition.calculationLink}
+                        options={this.state.calculationOptions}
+                        clearable={false}
+                        searchable={false} />
                     </FormGroup>
                     <FormGroup>
                       <FormLabel isRequired="true">{t(`rulesFlyout.condition.timePeriod`)}</FormLabel>
                       <FormControl
                         type="duration"
                         className="long"
-                        link={condition.timePeriod} />
+                        link={condition.durationLink} />
                     </FormGroup>
                     <FormGroup>
                       <FormLabel isRequired="true">{t(`rulesFlyout.condition.operator`)}</FormLabel>
                       <FormControl
                         type="select"
                         className="short"
                         placeholder={t(`rulesFlyout.condition.operatorPlaceholder`)}
-                        link={condition.operator} />
+                        link={condition.operatorLink}
+                        options={this.state.operatorOptions}
+                        value={this.state.operatorOptions[0].value}
+                        clearable={false}
+                        searchable={false} />
                     </FormGroup>
                     <FormGroup>
                       <FormLabel isRequired="true">{t(`rulesFlyout.condition.value`)}</FormLabel>
                       <FormControl
                         type="text"
                         placeholder={t(`rulesFlyout.condition.valuePlaceholder`)}
-                        link={condition.value} />
+                        link={condition.valueLink} />
                     </FormGroup>
+                    {
+                      conditionLinks.length > 1 &&
```

Comment:
The button should probably enable/disable instead of showing/hiding.

Reply:
I think otherwise. There is no use of displaying it when there is only one condition.

Reply:
fixed